From 5616015f548a9beda791d8d607e1b17ebdc1e09d Mon Sep 17 00:00:00 2001
From: kernel test robot <lkp@intel.com>
Date: Wed, 12 May 2021 09:24:37 +0900
Subject: [PATCH] cifsd: fix boolreturn.cocci warnings
Git-commit: 5616015f548a9beda791d8d607e1b17ebdc1e09d
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


fs/cifsd/smb2pdu.c:8098:8-9: WARNING: return of 0/1 in function
'smb2_is_sign_req' with return type bool

 Return statements in functions returning bool should use  true/false
instead of 1/0.
Generated by: scripts/coccinelle/misc/boolreturn.cocci

Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: kernel test robot <lkp@intel.com>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/smb2pdu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifsd/smb2pdu.c b/fs/cifsd/smb2pdu.c
index 18576148f530..e17ad2032fc7 100644
--- a/fs/cifsd/smb2pdu.c
+++ b/fs/cifsd/smb2pdu.c
@@ -7713,7 +7713,7 @@ bool smb2_is_sign_req(struct ksmbd_work *work, unsigned int command)
 	    command != SMB2_OPLOCK_BREAK_HE)
 		return true;
 
-	return 0;
+	return false;
 }
 
 /**
-- 
2.33.0

